linux_os/guide/system/apparmor/ensure_apparmor_enforce_or_complain/oval/shared.xml

```diff
@@ -0,0 +1,32 @@
+<def-group>
+  <definition class="compliance" id="ensure_apparmor_enforce_or_complain" version="1">
+    {{{ oval_metadata("Ensure all AppArmor Profiles are in var_set_apparmor_mode") }}}
```

Comment:
I think there is something wrong here. The final built content looks like
`<oval-def:description>Ensure all AppArmor Profiles are in var_set_apparmor_mode</oval-def:description>`.

Reply:
nice catch, I will fix it